src/prefect/new_task_engine.py

```diff
@@ -392,17 +408,16 @@ def handle_timeout(self, exc: TimeoutError) -> None:
             )
             self.set_state(state)
 
-    def handle_rollback(self, exc: RollBack, transaction: Transaction) -> None:
+    def handle_rollback(self, exc: RollBack) -> None:
         message = (
-            f"Task run raised rollback error, rolling back transaction {transaction}"
+            f"Task run raised RollBack error, rolled back transaction {transaction}"
```

Comment:
Was the removal of `transaction` in the signature accidental? It's still referenced here

Reply:
Whoa good catch, I'm surprised tests pass with that - yea removal was intentional, as the transaction now rolls itself back

Reply:
Oh it formats the `transaction` context manager 😅 - will fix